packages/ui/package.json

```diff
@@ -67,6 +67,10 @@
     "./Tab": {
       "types": "./dist/tab/Tab.svelte.d.ts",
       "svelte": "./dist/tab/Tab.svelte"
+    },
+    "./LinearProgress": {
+      "types": "./dist/spinner/LinearProgress.d.ts",
```

Comment:
could you rename the `spinner` directory to `progress` ?

Reply:
Done